README.md

```diff
@@ -21,6 +21,60 @@ Usage:
 - Run `quarto preview` to serve the site in Quarto's local webserver
 - Review the [`_quarto.yml`](_quarto.yml) file and [Github Actions workflow](.github/workflows/gh-pages.yml) for more details
 
+## Generating API reference documentation
+
+The repository includes an API documentation generator that takes documentation
+comments from Autonity protocol contracts and converts them into Markdown files.
+
+**TL/DR:** Symlink an [autonity](https://github.com/autonity/autonity) repository
+clone into the root of this repository and run [`apidocgen`](./_apidocgen/). The
+configuration is in [`config.toml`](./config.toml); the generated files are saved
+into `./contracts/<autonity-tag>/`.
+
+### Setup
+
+- Either clone the [Autonity Go Client](https://github.com/autonity/autonity) into the
+  root of this repository
+- Or add a symlink to an existing clone into this repository with `ln -s /path/to/autonity`
+
+### Adding API documentation for a new Autonity release
+
+1. If there are new contracts to document, add them to [`config.toml`](./config.toml).
+   The configuration is documented [here](./_apidocgen/README.md#configuration).
+1. Switch to the release tag in Autonity, e.g. for v0.14.0:
```

Comment:
```suggestion
2. Switch to the release tag in Autonity, e.g. for v0.14.0:
```

Reply:
From the [markdown reference](https://daringfireball.net/projects/markdown/syntax#list):
> It’s important to note that the actual numbers you use to mark the list have no effect on the HTML output Markdown produces. 

i.e. the list
```
1. one
2. two
3. three
```
produces the same output as
```
1. one
1. two
1. three
```
or
```
1. one
10. two
100. three
```
You can check it if you click the "rich diff" button in GitHub diff view.

Reply:
If you prefer having the right numbers, I wouldn't like to mess up the convention in the repo.